packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -63,3 +69,198 @@ describe("MafsGraph", () => {
         expect(mockChangeHandler).toHaveBeenCalled();
     });
 });
+
+function map(
+    num: number,
+    startMin: number,
+    startMax: number,
+    targetMin: number,
+    targetMax: number,
+): number {
+    return (
+        ((num - startMin) / (startMax - startMin)) * (targetMax - targetMin) +
+        targetMin
+    );
+}
+
+function graphToPixel(
+    num: number,
+    range: vec.Vector2 = [-10, 10],
+    fullPixelSpace: number = 400,
+) {
+    const [rangeMin, rangeMax] = range;
+    const halfPixelSpace = fullPixelSpace / 2;
+    return map(num, rangeMin, rangeMax, halfPixelSpace * -1, halfPixelSpace);
+}
+
+describe("MafsGraph", () => {
+    let userEvent: UserEvent;
+    beforeEach(() => {
+        userEvent = userEventLib.setup({
+            advanceTimers: jest.advanceTimersByTime,
+        });
+    });
+
+    it("renders", () => {
+        const mockDispatch = jest.fn();
+        const state: InteractiveGraphState = {
+            type: "segment",
+            hasBeenInteractedWith: true,
+            range: [
+                [-10, 10],
+                [-10, 10],
+            ],
+            snapStep: [0.5, 0.5],
+            coords: [
+                [
+                    [0, 0],
+                    [-7, 0.5],
+                ],
+            ],
+        };
+
+        const baseMafsGraphProps = getBaseMafsGraphProps();
+
+        render(
+            <MafsGraph
+                state={state}
+                dispatch={mockDispatch}
+                {...baseMafsGraphProps}
+            />,
+        );
+
+        const line = screen.getByTestId("movable-line__line");
+
+        expect(line).toBeInTheDocument();
+
+        // Map graph coordinates to SVG coordinates
+        const expectedX1 = graphToPixel(0);
+        const expectedY1 = graphToPixel(0) * -1;
+        const expectedX2 = graphToPixel(-7);
+        const expectedY2 = graphToPixel(0.5) * -1;
+        expect(line.getAttribute("x1")).toBe(expectedX1 + "");
+        expect(line.getAttribute("y1")).toBe(expectedY1 + "");
+        expect(line.getAttribute("x2")).toBe(expectedX2 + "");
+        expect(line.getAttribute("y2")).toBe(expectedY2 + "");
```

Comment:
Idea: could we configure the graph so one unit = one pixel? Then we wouldn't have to transform the `expected` coordinates.

Reply:
I personally like that this really approximates what a user would see.